presto-product-tests/src/main/resources/tempto-configuration.yaml

```diff
@@ -20,8 +20,11 @@ databases:
     table_manager_type: hive
     warehouse_directory_path: /user/hive/warehouse
     path: /user/hive/warehouse
-    analyze_immutable_tables: false
-    analyze_mutable_tables: false
+    inject_stats_for_immutable_tables: true
+    inject_stats_for_mutable_tables: true
+    metastore:
```

Comment:
Curious, why was this not needed before? Was it hard-coded in Tempto?

Reply:
There was no such things in previous version of tempto. it is used to inject data statistics into hive metastore (previously we tried to run `ANALYZE`, but it was not a good idea...)